src/snowflake/cli/plugins/nativeapp/errno.py

```diff
@@ -13,3 +13,4 @@
 # limitations under the License.
 
 APPLICATION_NO_LONGER_AVAILABLE = 93079
+APPLICATION_OWNS_EXTERNAL_OBJECTS = 93128
```

Comment:
Should we just have a single errno file? Anything under API is technically consumable by client code, so IMHO it would make sense not to hide the NA-specific codes.

Reply:
yeah if we can get shared ownership of api/errno.py i'd move them there

Reply:
I don't see why not... Just make the change to CODEOWNERS in this PR and ask CLI folks to approve (you need their approval for this anyway).

Reply:
although that kinda breaks the pseudo-plugin architecture we've got going on